test/relations.test.js

```diff
@@ -709,40 +710,142 @@ describe('relations', function() {
       }
     });
 
-    it('should fetch scoped instances with paging filters', function(done) {
-      Physician.create(function(err, physician) {
-        physician.patients.create({name: 'a'}, function() {
-          physician.patients.create({name: 'z'}, function() {
-            physician.patients.create({name: 'c'}, function() {
-              verify(physician);
+    describe('fetch scoped instances with paging filters', function() {
+      var samplePatientId;
+      var physician;
+
+      beforeEach(createSampleData);
+
+      context('with filter skip', function() {
+        it('skips the first patient', function(done) {
+          physician.patients({skip: 1}, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(2);
+            ch[0].name.should.eql('z');
+            ch[1].name.should.eql('c');
+            done();
+          });
+        });
+      });
+      context('with filter order', function() {
+        it('orders the result by patient name', function(done) {
+          physician.patients({order: 'name DESC'}, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(3);
+            ch[0].name.should.eql('z');
+            ch[2].name.should.eql('a');
+            done();
+          });
+        });
+      });
+      context('with filter limit', function() {
+        it('limits to 1 result', function(done) {
+          physician.patients({limit: 1}, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(1);
+            ch[0].name.should.eql('a');
+            done();
+          });
+        });
+      });
+      context('with filter fields', function() {
+        it('includes field \'name\' but not \'age\'', function(done) {
+          var fieldsFilter = {fields: {name: true, age: false}};
+          physician.patients(fieldsFilter, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            should.exist(ch[0].name);
+            ch[0].name.should.eql('a');
+            should.not.exist(ch[0].age);
+            done();
+          });
+        });
+      });
+      context('with filter include', function() {
+        it('returns physicians inluced in patient', function(done) {
+          var includeFilter = {include: 'physicians'};
+          physician.patients(includeFilter, function(err, ch) {
+            should.not.exist(err);
+            ch.should.have.lengthOf(3);
+            should.exist(ch[0].physicians);
```

Comment:
nitpick: maybe check if `physicians` has current physician instance? (I might have misunderstood, just asking if we could do more checks on what `physicians` has).